definitions/procedures/remote_execution/remove_existing_settingsd.rb

```diff
@@ -0,0 +1,17 @@
+module Procedures::RemoteExecution
+  class RemoveExistingSettingsd < ForemanMaintain::Procedure
+    metadata do
+      param :dirpath,
+            'Directory path for settings.d folder',
+            :required => true
+      description 'Remove existing settings.d directory before installer run'
```

Comment:
I would add also, that `The next run of the installer will re-create the directory properly`